notebooks/TCO_calculator.ipynb

```diff
@@ -0,0 +1,807 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f3e4c571-624d-4db6-b4d9-ae912879967b",
+   "metadata": {},
+   "source": [
+    "# AIPerf -> LLM TCO Calculator Data Connector\n",
+    "\n",
+    "This notebook shows you how to do LLM performance benchmarking with the NVIDIA AIPerf tool and then export the data to a TCO (Total Cost of Ownership) calculator in [Excel spreadsheet format](./LLM_TCO_Calculator.xlsx).\n",
+    "\n",
+    "\n",
+    "To execute this notebook, you can use the NVIDIA Triton server container:\n",
+    "```\n",
+    "docker run --gpus=all --ipc=host --net=host --rm -it -v $PWD:/myworkspace nvcr.io/nvidia/tritonserver:25.05-py3-sdk bash  \n",
```

Comment:
This triton container is a bit old already. Is this something that is actively tested and updated regularly OR does this need to be pinned?

Reply:
not at all. It's just the container I used for development and have not yet tested on newer versions, like 25.10 etc... 

I think it should work but would appreciate if someone can go through the tutorial as a QA test with new containers.